packages/fx-core/src/plugins/resource/apim/index.ts

```diff
@@ -38,12 +38,17 @@ export class ApimPlugin implements Plugin {
   ): Promise<Result<QTreeNode | undefined, FxError>> {
     return await this.executeWithFxError(PluginLifeCycle.GetQuestions, _getQuestions, ctx, stage);
   }
-  
+
   public async getQuestionsForUserTask(
     func: Func,
     ctx: PluginContext
   ): Promise<Result<QTreeNode | undefined, FxError>> {
-    return await this.executeWithFxError(PluginLifeCycle.GetQuestions, _getQuestionsForUserTask, ctx, func);
+    return await this.executeWithFxError(
+      PluginLifeCycle.GetQuestions,
```

Comment:
Is this supposed to be `PluginLifeCycle.GetQuestionsForUserTask`?
If not, seems we can remove `GetQuestionsForUserTask` in the `PluginLifeCycle` definition

Reply:
Yes, it should be `PluginLifeCycle.GetQuestionsForUserTask`